go.mod

```diff
@@ -10,15 +10,16 @@ require (
 	github.com/docker/docker v1.4.2-0.20190924003213-a8608b5b67c7
 	github.com/docker/go-connections v0.4.0
 	github.com/golang/mock v1.3.1
-	github.com/google/go-cmp v0.3.0
+	github.com/golangci/golangci-lint v1.23.6 // indirect
+	github.com/google/go-cmp v0.4.0
 	github.com/google/go-containerregistry v0.0.0-20191018211754-b77a90c667af
 	github.com/heroku/color v0.0.6
 	github.com/mitchellh/ioprogress v0.0.0-20180201004757-6a23b12fa88e
-	github.com/onsi/gomega v1.7.0
+	github.com/onsi/gomega v1.8.1
 	github.com/pkg/errors v0.8.1
 	github.com/sclevine/spec v1.4.0
 	github.com/spf13/cobra v0.0.5
-	golang.org/x/crypto v0.0.0-20190510104115-cbcb75029529
+	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550
```

Comment:
Probably not necessary, but `make test` seemed to generate this change; figure it didn't hurt to update.